diff -urN uboot-tools-2024.04.orig/Makefile uboot-tools-2024.04/Makefile
--- uboot-tools-2024.04.orig/Makefile	2024-08-07 09:37:59.824741018 +0200
+++ uboot-tools-2024.04/Makefile	2024-08-07 09:38:40.815429183 +0200
@@ -532,7 +532,7 @@
 
 no-dot-config-targets := clean clobber mrproper distclean \
 			 help %docs check% coccicheck \
-			 ubootversion backup tests check pcheck qcheck tcheck \
+			 ubootversion backup tools-only tests check pcheck qcheck tcheck \
 			 pylint pylint_err _pip pip pip_test pip_release
 
 config-targets := 0
